gravitee-am-gateway/gravitee-am-gateway-handler/gravitee-am-gateway-handler-common/src/main/java/io/gravitee/am/gateway/handler/common/vertx/web/endpoint/ErrorEndpoint.java

```diff
@@ -42,6 +42,7 @@
 import java.util.HashMap;
 import java.util.Map;
 
```

Comment:
Few unused import, not added by you. Would be great if you could clean them as well :)